src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/VariableDeclarations.cs

```diff
@@ -112,6 +146,7 @@ static void AppendVariableDeclarations(ImmutableArray<LocalDeclarationStatementS
             {
                 (string managed, string native) = context.GetIdentifiers(marshaller.TypeInfo);
 
+                var info = marshaller.TypeInfo;
```

Comment:
Is this needed to be a local? It is fine if it is but there are a bunch of places in this method that use `marshaller.TypeInfo`, is it the same?